app/services/availability_service.py

```
@@ -39,78 +40,194 @@
+            int: 00:00으로부터 경과된 '분' (예: "14:00" -> 840)
+            
+        Raises:
+            ValueError: 형식이 맞지 않거나, 분 단위가 0이 아닐 경우
+            
+        Rationale (의도):
+            문자열 기반의 타임라인 계산 시 대소 비교나 시간 간격 연산의 복잡도가 증가하므로,
+            수치형 차원(Int) 연산으로 정규화하여 안정성을 높임.
+        """
+        match = re.fullmatch(r"(\d{2}):(\d{2})", slot)
+        if not match:
+            raise ValueError(f"?쒓컙 ?뺤떇???щ컮瑜댁? ?딆뒿?덈떎: {slot}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)